packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -17,7 +17,49 @@ export const MenuItemTemplate = html<MenuItem>`
         class="${x => (x.disabled ? "disabled" : "")} ${x =>
             x.expanded ? "expanded" : ""}"
     >
-        ${startTemplate}
+        <div part="start-container" class="start-container">
```

Comment:
What happens when I want an icon in addition to this? I do not think this should replace start/end.

Reply:
`${startTemplate}` is still there. I put it in the same div container as the radio and checkbox inputs.

Reply:
@chrisdholt even though I still have `${startTemplate}` alongside the radio and checkbox inputs, is this still not the right approach, or was there another concern with this change?